### DISCUSSION TITLE
Allow filtering by repositories and/or organizations

### DISCUSSION THREAD

**Discussion by eamodio** [community/community#3007] [Mobile]
Not having notification filtering by repositories makes notification triaging extremely difficult.

I would additionally like to have filtering by organizations -- so that for example, while on vacation, I can completely filter out all my work notifications and focus solely on my OSS ones.

**Comment by becca** [comment]
You can filter your notification list by repo(s) as mentioned in our documentation [here](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/managing-notifications-from-your-inbox#supported-queries-for-custom-filters). The organization filter is not yet supported but it's a request the team is highly aware of and the public roadmap is a good place to follow for more updates.

This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in ur [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).

**Reply by eamodio** [reply]
This feedback is for the GitHub Mobile app (I'm sure I picked the Mobile Feedback category when I created this issue). The documentation you linked is for the web, and doesn't help in the mobile app at all.
